apps/expo/src/components/TotalAndSortHeader.tsx

```diff
@@ -11,12 +11,12 @@ interface Props {
 export const TotalAndSortHeader: FC<Props> = ({ total, tab, filter }) => {
   return (
     <View className="mt-4 w-full flex-row items-end justify-between">
-      <View>
+      <View className="mx-4">
         <Text className=" font-nunito-bold text-xl">
           {total} {tab}
         </Text>
       </View>
-      <View className="flex-row gap-2">
+      <View className="mx-2.5 flex-row gap-2">
```

Comment:
Same as previous comment

Reply:
Already checked with library rendering. Wala na problems all good. Mu na na dugayan ko ay gina double check ko if ok pag render both sa library and profile